src/codegen/stackvm/codegen_stackvm.cc

```diff
@@ -470,8 +470,8 @@ void CodeGenStackVM::VisitExpr_(const Select *op) {
 }
 
 void CodeGenStackVM::VisitStmt_(const AssertStmt *op) {
-  if (op->message.as<StringImm>()) {
-    int sid = this->GetStrID(op->message.as<StringImm>()->value);
+  if (auto str = op->message.as<StringImm>()) {
```

Comment:
Make this "const auto* str" instead of "auto str", to expressly show that str is expected to be a const pointer.

Reply:
The "as" template is defined here. Very hard to dig out:
https://github.com/dmlc/HalideIR/blob/master/src/tvm/node/node.h#L306

Reply:
Fixed.